packages/tiny-router/package.json

```diff
@@ -23,7 +23,8 @@
   "dependencies": {
     "@frontity/router": "^1.1.2",
     "@frontity/source": "^1.4.0",
-    "frontity": "^1.13.0"
+    "frontity": "^1.13.0",
+    "query-string": "^6.13.7"
```

Comment:
The `wp-source` already has `query-string` as a dependency so this is not adding to the bundle size.